components/brave_vpn/brave_vpn_os_connection_api_sim.cc

```diff
@@ -49,14 +49,19 @@ void BraveVPNOSConnectionAPISim::Connect(const std::string& name) {
         FROM_HERE,
         base::BindOnce(&BraveVPNOSConnectionAPISim::OnConnected,
                        weak_factory_.GetWeakPtr(), name, true),
-        base::TimeDelta::FromSeconds(3));
+        base::TimeDelta::FromSeconds(1));
```

Comment:
is there a chance this could leak to flaky tests?

Reply:
We don't use this sim api for test.